index.css

```diff
@@ -0,0 +1,747 @@
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 20px;
+    text-align: center;
+    letter-spacing: 0.2px;
+    color: #737373;
+}
+
+.page3__body{
+    padding-top: 200px;
+}
+
+.page3__body-bg{
+    padding-top: 50px;
+    display: block;
+    width: 100%;
+    height: 330px;
+    background: #006b5b;
+}
+
+.page3__body-img{
+    position: absolute;
+    left: 200px;
+    top: 2120px;
+}
+
+
+.page4{
+    padding-top: 100px;
+    background: #FAFAFA;
+}
+
+.page4__body{
+    display: flex;
+    flex-direction: column;
+    gap: 50px;
+    align-items: center;
+}
+
+.page4__heading{
+    padding-left: 130px;
+    width: 476px;
+height: 20px;
+font-family: 'Montserrat';
+font-style: normal;
+font-weight: 700;
+font-size: 40px;
+line-height: 57px;
+letter-spacing: 0.2px;
+color: #252B42;
+
+}
+
+.page4__para{
+    width: 470px;
+height: 40px;
+font-family: 'Montserrat';
+font-style: normal;
+font-weight: 500;
+font-size: 14px;
+line-height: 20px;
+text-align: center;
+letter-spacing: 0.2px;
+color: #737373;
+
+}
+
+.page4__cards{
+    display: flex;
+    gap: 30px;
+    padding-left: 50px;
+
+}
+
+
+
+.card4{
+    display: block;
+    background: #FFFFFF;
+border: 1px solid #DEDEDE;
+border-radius: 5px;
+width: 327px;
+height: 244px;
+margin-top: 70px;
+margin-left: 30px;
+margin-bottom: 150px;
+}
+
+.card4__content{
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    padding-left: 40px;
+    padding-bottom: 20px;
+}
+
+.header{
+    display:flex;
+    padding-top: 20px;
+    gap: 5px;
+
+}
+.card4__para{
+    width: 234px;
+height: 72px;
+font-family: 'Montserrat';
+font-style: normal;
+font-weight: 600;
+font-size: 14px;
+line-height: 24px;
+letter-spacing: 0.2px;
+color: #737373;
+}
+
+.footer{
+    display: flex;
+    gap: 15px;
+}
+
+.footer__image{
+    width: 50px;
+    height: 50px;
+}
+
+.footer__name{
+    width: 105px;
+height: 10px;
+font-family: 'Montserrat';
+font-style: normal;
+font-weight: 600;
+font-size: 14px;
+line-height: 24px;
+letter-spacing: 0.2px;
+color: #FFA62B;
+
+}
+
+.footer__caption{
+    width: 67px;
+height: 14px;
+font-family: 'Montserrat';
+font-style: normal;
+font-weight: 600;
+font-size: 14px;
+line-height: 24px;
+letter-spacing: 0.2px;
+color: #252B42;
+
+}
+
+.footer__words{
+    display: flex;
+    flex-direction: column;
+    gap: 15px;
+}
+
+
+
+.page5__head{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 40px;
+    padding-top: 80px;
+}
+
+.page5__heading{
+
+width: 350px;
+height: 27px;
+font-family: 'Montserrat';
+font-style: normal;
+font-weight: 700;
+font-size: 40px;
+line-height: 57px;
+letter-spacing: 0.2px;
+color: #252B42;
+
+}
+
+.page5__para{
+width: 470px;
+height: 40px;
+font-family: 'Montserrat';
+font-style: normal;
+font-weight: 500;
+font-size: 14px;
+line-height: 20px;
+text-align: center;
+letter-spacing: 0.2px;
+color: #737373;
+}
+
+
+.cards{
```

Comment:
`cards ` and `card1` and also ` card` the names are the same only the deference is `s` `1` name them very differently so that they can completely stand by  themselves with their names